addons/point_of_sale/static/src/js/devices.js

```diff
@@ -434,6 +436,23 @@ var ProxyDevice  = core.Class.extend(core.mixins.PropertiesMixin,{
         send_printing_job();
     },
 
+    print_sale_details: function() { 
+        var self = this;
+        new Model('report.pos.details').call('get_sale_details').then(function(result){
```

Comment:
Should be `report.point_of_sale.report_saledetails` instead of `report.pos.details` I think.